scripts/cli/cli_helper.py

```diff
@@ -26,12 +28,46 @@ def parse_source() -> List[str]:
     Returns:
         List[str]: A list of paths.
     """
-    parser = argparse.ArgumentParser()
-    parser.add_argument("--source", dest="source", nargs="+", required=True)
-    arguments = parser.parse_args()
+    parse = argparse.ArgumentParser()
+    parse.add_argument("--source", dest="source", nargs="+", required=True)
+    arguments = parse.parse_args()
 
     return format_source(arguments.source)
 
 
 def is_argo() -> bool:
     return bool(environ.get("ARGO_TEMPLATE"))
+
+
+def format_date(date: datetime) -> str:
```

Comment:
It would be a good idea to have some unit tests for these formatters and converters